components/ui/ModerationChecklist.vue

```diff
@@ -0,0 +1,1042 @@
+In accordance with section 2.2 of [Modrinth's Content Rules](https://modrinth.com/legal/rules) we ask that you provide a text alternative to your current Description. It is important that your Description contains enough detail about your project that a user can have a full understanding of it from text alone. A text-based transcription allows for those using screen readers, and users with slow internet connections unable to load images to be able to access the contents of your Description. This also acts as a backup in case the image in your Description ever goes offline for some reason.
+We appreciate how much effort you put into your Description, but accessibility is important to us at Modrinth, if you would like you could put the transcription of your Description entirely in a \`details\` tag, so as to not spoil the visuals of your Description.`,
+        },
+        {
+          name: 'Non-standard text',
+          resultingMessage: `## Description Accessibility
+Per section 2 of [Modrinth's Content Rules](https://modrinth.com/legal/rules#clear-and-honest-function) your description must be plainly readable and accessible. Using non-standard text characters like Zalgo or "fancy text" in place of text anywhere in your project, including the Description, Summary, or Title can make your project pages inaccessible. This is important for users who rely on Screen Readers and for search engines in order to provide relevant results to users. Please remove any instances of this type of text.`,
+        },
+      ],
+    },
+    {
+      id: 'links',
+      question: `Are the project's links accessible and not misleading?`,
+      shown:
+        props.project.issues_url ||
+        props.project.source_url ||
+        props.project.wiki_url ||
+        props.project.discord_url ||
+        props.project.donation_urls.length > 0,
+      rules: [
+        `All links must be accessible.`,
+        `All links must correspond correctly with a label (ex: Discord link should not go to a YouTube channel)`,
+      ],
+      options: [
+        {
+          name: 'Links are misused',
+          resultingMessage: `## Misuse of External Resources
+Per section 5.4 of [Modrinth's Content Rules](https://modrinth.com/legal/rules#miscellaneous) all links must lead to correctly labeled publicly available resources that are directly related to your project.`,
+        },
+        {
+          name: 'Not accessible (source)',
+          resultingMessage: `## Unreachable Links
+Per section 5.4 of [Modrinth's Content Rules](https://modrinth.com/legal/rules#miscellaneous) all links must lead to correctly labeled publicly available resources that are directly related to your project.
+Currently, your Source link directs to a Page Not Found error, likely because your repository is private, make sure to make your repository public before resubmitting your project!`,
+          fillers: [
+            {
+              id: 'EXPLAINER',
```

Comment:
This looks like a bug. `EXPLAINER` is not used in the above message.

Reply:
thanks!